### DISCUSSION TITLE
The ability to create perfect inclined/declined ramps on the terrain object.

### DISCUSSION THREAD

**Discussion by antzGames** [JamesTKhan/Mundus#168] [Ideas]
The ability to create perfect inclined/declined ramps on the terrain object.

 In addition having a SQUARE terrain brush that raises/lowers terrain for the entire square brush size.  Both requests are related.

It is incredibly hard to create flat inclined/declined terrain with the current terrain editor brushes and tools.

**Comment by JamesTKhan** [comment]
Both of these should be doable. A square brush is a bit odd for traditional terrain but I am not opposed to it and I believe its relatively simple to add new brush shapes albeit I haven't tried yet. Would the square brush have both a solid (hard edge) version, and soft edge version like the circle?

I like the idea for the ramp tool as well.

**Reply by antzGames** [reply]
> Would the square brush have both a solid (hard edge) version, and soft edge version like the circle?

At a minimum hard edge.  But both would be great.

From: https://github.com/blackears/blenderTerrainSculpt

**Ramp**

Shortcut R.

Click and drag from one location to another to create a ramp between those locations.

![image](https://github.com/JamesTKhan/Mundus/assets/10563814/d94b0a19-c6f7-4ce2-b83c-b5d2017d7e85)
